web/templates/compose.hbs

```diff
@@ -12,26 +12,25 @@
     </div>
     <div id="compose_controls" class="new-style">
         <div id="compose_buttons">
-            <span class="new_message_button reply_button_container" data-tooltip-template-id="compose_reply_message_button_tooltip_template">
-                <button type="button" class="button small rounded compose_reply_button"
-                  id="left_bar_compose_reply_button_big">
+            <div class="new_message_button reply_button_container">
+                <button type="button" class="button small compose_reply_button"
+                  id="left_bar_compose_reply_button_big"
```

Comment:
Probably not worth adding more churn to this PR, but let's plan to finish purging the puzzling `left_bar_` names from this file -- I have no idea what `left_bar` ever referred to.

Reply:
Yes. I've already got that as a todo--it's a little more extensive to clean up, so it's even more churn than meets the eye.